core/src/main/java/jenkins/security/ClassFilterImpl.java

```diff
@@ -106,7 +104,7 @@ private static void mockOff() {
         ClassFilter.setDefault(ClassFilter.NONE); // even Method on the standard blacklist is going to explode
     }
 
-    @VisibleForTesting
+    //@VisibleForTesting
```

Comment:
To be consistent with the rest of this PR:

```suggestion
    @VisibleForTesting
```